Services/Twilio/Rest/Addresses.php

```diff
@@ -3,4 +3,19 @@
 class Services_Twilio_Rest_Addresses
     extends Services_Twilio_ListResource
 {
+
+	public function create($customerName, $street, $city, $region, $postalCode, $isoCountry,$friendlyName = null) {
+		$params = array(
+				'CustomerName' => $customerName,
+				'Street' => $street,
+				'City' => $city,
+				'Region' => $region,
+				'PostalCode' => $postalCode,
+				'IsoCountry' => $isoCountry,
+		);
+		if (!is_null($friendlyName)) {
```

Comment:
Due to false evaluation in PHP I think taking anything specified by the user is prudent. If someone wants to name their Address "0" (zero) they should be able to.